pack/dwl/wda.json

```diff
@@ -226,6 +226,7 @@
         "sanity": 1,
         "skill_agility": 1,
         "slot": "Body",
+        "tags": "pa.",
```

Comment:
was it cleared up whether the parley access is limited to _**Parley.**_ actions or does it also apply to cards like this?

In this specific case, it does not make a functional difference though as `Neutral 0-5` grants access anyway.

Reply:
I believe I got a clarification that it applies to parley generally -- though not on a card like Grizzled (which has Parley in italics as a rule reminder)

Reply:
As far as I know though, it doesn't actually matter yet (all the cards that mention parley are already rogue IIRC)